.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,34 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/javascript-node
+{
+  "name": "Node.js",
+  // Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+  "image": "mcr.microsoft.com/devcontainers/javascript-node:1-20-bookworm",
```

Comment:
When I try to use the node 22 image, I can get following error:

![image](https://github.com/user-attachments/assets/e8516153-b8f6-47ce-95e5-77fb5a3ec2dd)

Reply:
Is there a particular reason for picking this image? I suppose it doesn't make much difference for an application like this one with basically zero "system" dependencies beyond node, but my instinct would be to base this off be `node:20-alpine` since that is what we use in https://github.com/badges/shields/blob/master/Dockerfile

In any case, yes. We definitely want node 20

Reply:
vscode can generate a devcontainer which then uses Microsoft's set of "officially maintained" ones. That's where this image comes from. I can use a basic `node:20-alpine` as well, the devcontainer config will just need to also prep git and some other things. Given it's "the node image designed for devcontainers" managed by Microsoft, would you be fine leaving it in? Happy to change it to `node:20-alpine` and appropriate other changes to the devcontainer if you'd prefer.

Reply:
yeah that makes sense